src/main/java/com/purbon/kafka/topology/roles/Principal.java

```diff
@@ -0,0 +1,20 @@
+package com.purbon.kafka.topology.roles;
```

Comment:
not sure this is the best package for this, what about some place under model?

Reply:
Moved the `Principal` class under `model.users`